.spi.yml

```diff
@@ -5,7 +5,9 @@ builder:
     scheme: SpotifyAPI
   - platform: ios
     scheme: SpotifyAPI
-  - platform: macos
+  - platform: macos-xcodebuild
+    scheme: SpotifyAPI
+  - platform: macos-xcodebuild-arm
```

Comment:
Note that there are two other macOS "platforms":

- `macos-spm`
- `macos-spm-arm`

but these don't require any special casing, because as opposed to `xcodebuild`, SPM knows how to build the package and doesn't need schemes.